CRYOSMSApp/src/CRYOSMSDriver.cpp

```diff
@@ -852,11 +1180,20 @@ void CRYOSMSDriver::continueAbort()
 extern "C"
 {
 
-	int CRYOSMSConfigure(const char *portName, std::string devPrefix)
+	int CRYOSMSConfigure(const char *portName, std::string devPrefix, const char *TToA, const char *writeUnit, const char *displayUnit, const char *maxCurr, const char *maxVolt,
+		const char *allowPersist, const char *fastFilterValue, const char *filterValue, const char *npp, const char *fastPersistentSettletime, const char *persistentSettletime,
+		const char *fastRate, const char *useSwitch, const char *switchTempPv, const char *switchHigh, const char *switchLow, const char *switchStableNumber, const char *heaterTolerance,
+		const char *switchTimeout, const char *heaterOut, const char *useMagnetTemp, const char *magnetTempPv, const char *maxMagnetTemp,
+		const char *minMagnetTemp, const char *compOffAct, const char *noOfComp, const char *minNoOfComp, const char *comp1StatPv, const char *comp2StatPv, const char *rampFile)
+
 	{
 		try
 		{
-			new CRYOSMSDriver(portName, devPrefix);
+			new CRYOSMSDriver(portName, devPrefix, TToA, writeUnit, displayUnit, maxCurr, maxVolt,
+				allowPersist, fastFilterValue, filterValue, npp, fastPersistentSettletime, persistentSettletime,
+				fastRate, useSwitch, switchTempPv, switchHigh, switchLow, switchStableNumber, heaterTolerance,
+				switchTimeout, heaterOut, useMagnetTemp, magnetTempPv, maxMagnetTemp,
+				minMagnetTemp, compOffAct, noOfComp, minNoOfComp, comp1StatPv, comp2StatPv, rampFile);
 			return asynSuccess;
```

Comment:
Would it be possible to create the map in  CRYOSMSConfigure and pass to CRYOSMSDriver ? This avoid the duplication of arguments and possible ordering errors.